nano-zkp/src/lib.rs

```diff
@@ -1,5 +1,3 @@
-#![allow(dead_code)]
```

Comment:
Looks like this generated warnings of unused elements in `nano-zkp/src/gadgets/mnt4/merkle_tree.rs`, `nano-zkp/src/gadgets/mnt6/serialize.rs` and  `nano-zkp/src/gadgets/mnt6/y_to_bit.rs`. Could you check them?

Reply:
Fixed by marking those modules as `#[allow(dead_code)]` since I'd guess that we want to use them at some point.